v1/python-sdk/tutorials/automl-with-azureml/forecasting-demand-forecasting-many-models/auto-ml-forecasting-demand-forecasting-many-models.ipynb

```diff
@@ -0,0 +1,1626 @@
+   "metadata": {},
+   "source": [
+    "## 5. Train the baseline model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use Naive model as our baseline. To train it, we kick off another automl experiment with the following settings. Please note that we added the Naive model to the allowed models list, and set the number of iterations to 1 since we only interested in training one specific model. To reduce the training time, we set the number of cross validations to 2. Read the following [document](https://learn.microsoft.com/en-us/azure/machine-learning/v1/how-to-auto-train-forecast-v1#training-and-validation-data) for more information on this topic.\n",
+    "\n",
+    "The only `AutoMLConfig` settings you might consider changing are the `experiment_timeout_hours` and `allowed_models`. You might want to increase the experiment timeout if your data has lots of unique time series. The allowed model list can be modified to refect a different choice of the baseline model and can be selected from the supported [forecasting models](https://learn.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.forecasting) and [regression models](https://learn.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.constants.supportedmodels.regression)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.automl.core.forecasting_parameters import ForecastingParameters\n",
+    "\n",
+    "forecasting_parameters = ForecastingParameters(\n",
+    "    time_column_name=time_column_name,\n",
+    "    forecast_horizon=forecast_horizon,\n",
+    "    time_series_id_column_names=time_series_id_column_names,\n",
+    "    cv_step_size=1,\n",
+    "    freq=\"H\",\n",
+    ")\n",
+    "\n",
+    "automl_config = AutoMLConfig(\n",
+    "    task=\"forecasting\",\n",
+    "    debug_log=\"baseline.log\",\n",
+    "    primary_metric=\"normalized_root_mean_squared_error\",\n",
+    "    experiment_timeout_hours=1,\n",
+    "    iterations=1,\n",
+    "    training_data=train_dataset,\n",
+    "    label_column_name=target_column_name,\n",
+    "    compute_target=compute_target,\n",
+    "    enable_early_stopping=True,\n",
+    "    n_cross_validations=2,\n",
+    "    verbosity=logging.INFO,\n",
+    "    max_cores_per_iteration=-1,\n",
+    "    enable_dnn=False,\n",
+    "    allowed_models=[\"Naive\"],\n",
+    "    forecasting_parameters=forecasting_parameters,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Is the baseline model trained? {IS_BASE_MODEL_TRAINED}\\n---\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IS_BASE_MODEL_TRAINED = False"
```

Comment:
Probably, we do not need this line, instead, let us say: If model was already trained, let us skip training and use the results from already trained models. We will need the run ID of an already ran pipeline to access them.

Reply:
fixed. It was redundant.